packages/core/admin/ee/server/controllers/workflows/stages/index.js

```diff
@@ -137,6 +137,16 @@ module.exports = {
 
     const { model_uid: modelUID, id } = ctx.params;
 
+    if (
+      strapi
+        .plugin('content-manager')
+        .service('permission-checker')
+        .create({ userAbility: ctx.state.userAbility, model: modelUID })
+        .cannot.read()
+    ) {
+      return ctx.forbidden();
+    }
```

Comment:
nice! could you add the same for the updateStages endpoint? so they can not be updated if there is no update permission for the content type?

Reply:
Do we definitely want to do that now? I'd left a comment on the JIRA ticket to confirm with product

CONTENT-1831

Reply:
Makes sense , lets move this forward then 😌 you are right